docs/howTo/requiredSoftware.md

```diff
@@ -0,0 +1,113 @@
+<style>
```

Comment:
Not sure if this is just rendering incorrectly in GitHub - but the how to preview linked in the PR description looks like this to me:
![image](https://github.com/alphagov/di-mobile-android-onelogin-app/assets/109979772/ccc297cd-e701-4180-bdb3-117bb6baf8c9)

Reply:
Oooh, weird... inline styles it is then!